src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -93,10 +97,31 @@ public RestoreCommand(FrameworkName fallbackFramework)
 
             foreach (var category in ErrorMessages)
             {
-                Reports.Error.WriteLine("Errors in {0}".Red().Bold(), category.Key);
+                Reports.Error.WriteLine("{0}Errors in {1}".Red().Bold(), Environment.NewLine, category.Key);
```

Comment:
Ugh, we really need to unify this coloring and how errors are displayed. It seems that everyone does it in their own way...